vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -76,9 +80,8 @@ The _app-builder_ is customized as part of many Entando implementations.
 It can be customized at runtime via micro frontends 
 [widget configuration](../../tutorials/create/mfe/widget-configuration.md).
 
-## component-manager
-The _component-manager_ provides the link between the entando-de-app (or your custom core instance) and the 
-Entando Component Repository (ECR). The _component-manager_ queries the entando-k8s service to fetch available 
+## component-manager(cm)
+The _component-manager_ provides the link between the entando-de-app (or your custom core instance) and the ECR. The _component-manager_ queries the entando-k8s service to fetch available 
 bundles that have been deployed as custom resources inside of an Entando cluster. 
 The _component-manager_ also manages the relationships between an Entando application and the 
 installed plugins. This can be seen in the plugin link custom resources in Kubernetes. 
```

Comment:
"This can be seen in the plugin link custom resources in Kubernetes" -- at the plugin link to custom resources? don't know what this sentence means; very fragmented